portmidi/pm_linux/pmlinuxalsa.c

```diff
@@ -32,9 +32,9 @@
 #endif
 
 /* to store client/port in the device descriptor */
-#define MAKE_DESCRIPTOR(client, port) ((void*)(((client) << 8) | (port)))
-#define GET_DESCRIPTOR_CLIENT(info) ((((int)(info)) >> 8) & 0xff)
-#define GET_DESCRIPTOR_PORT(info) (((int)(info)) & 0xff)
+#define MAKE_DESCRIPTOR(client, port) ((void*)(size_t)(((client) << 8) | (port)))

```

Comment:
I'm curious, what was the warning in that case if you remember it?

Reply:
Something like this `warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]`

Reply:
Oh ok, I see. I was troubled by the double cast, but now I see how it makes sense.